### DISCUSSION TITLE
Wording Suggestion - Successfully set new credentials

### DISCUSSION THREAD

**Discussion by bgribaudo** [microsoft/vscode-powerquery-sdk#185] [Ideas]
After setting a new credential, the toast notification that pops up reports that a new credential was "generated". This phraseology is interesting, as no credentials were actually _generated_, but rather one was _set_. 

In contrast, the fact that _setting_ was what occurred is reflected in the output on the command line.

![image](https://user-images.githubusercontent.com/857202/197591606-757e0185-c107-401f-afa4-82f126f72ddf.png)

Suggestion: Revise this toast notification from "...has been generated successfully" to "...has been set." Maybe also trim off the word "New" in front of the message, as arguably it is implied so is redundant.

**Comment by ptyx507x** [comment]
What about "Credential has been set (Kind: Anonymous)" ?

**Reply by bgribaudo** [reply]
Something like that would be great.
